README.md

```diff
@@ -31,19 +31,34 @@ General instructions for creating a GitHub OAuth token can be found [here](https
+| EnvironmentType  | Optional  | Environment type used by AWS CodeBuild. See [the documentation](https://docs.aws.amazon.com/codebuild/latest/userguide/create-project.html#create-project-cli) for details. Default: LINUX_CONTAINER |
+| BuildSpecFilePath  | Optional  | CodeBuild build spec file name for build stage. See [Build Specification Reference for CodeBuild](https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html). Default: buildspec.yaml |
+| IntegTestRoleName  | Optional  | IAM role name for test stage. This role needs to be configured to allow codebuild.amazonaws.com and cloudformation.amazonaws.com to assume it. Test stage will not be added if default value is used. Default: '' |
+| IntegTestBuildSpecFilePath  | Optional  | CodeBuild build spec file name for test stage. This parameter is only used if you provide the `IntegTestRoleName` parameter. See [the documentation](https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html). Default: integ-test-buildspec.yaml |
+| DeployRoleName  | Optional  | IAM role name for deploy stage. This role needs to be configured to allow cloudformation.amazonaws.com to assume it. Deploy stage will not be added if default value is used. Default: '' |
+| DeployStackName  | Optional  | CloudFormation stack name for deploy stage. Default: ''. This parameter is only used if you provide the `DeployRoleName` parameter. Note that if you provide the `DeployRoleName` but do not provide a `DeployStackName` then AWS CodePipeline will fail. |
+| DeployParameterOverrides  | Optional  | CloudFormation parameter overrides for deploy stage in JSON string. For more information and an example, see the `ParameterOverrides` parameter of [AWS CloudFormation Configuration Properties Reference](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/continuous-delivery-codepipeline-action-reference.html#w2ab1c13c13b9). Default: {} |
+| PublishToSAR  | Optional  | Boolean to indicate whether or not include publish stage. Allowed values: true, false. Default: false |
+
+The following parameters are only applicable if `SourceCodeProvider` is GitHub.
+
+| Parameter     | Required      | Description   |
+| ------------- | ------------- | ------------- |
+| GitHubOwner  | Optional  | GitHub username owning the repo. |
+| GitHubRepo  | Optional  | GitHub repo name (just the name, not the full URL). |
+| GitHubOAuthToken  | Optional  | OAuth token used by AWS CodeBuild to connect to GitHub. |
+| GitHubBranch  | Optional  | GitHub repo branch name. Default: master. |
+
+The following parameters are only applicable if `SourceCodeProvider` is CodeCommit.
+
+| Parameter     | Required      | Description   |
+| ------------- | ------------- | ------------- |
+| CodeCommitRepoName  | Optional  | CodeCommit repository name (just the name, not the full URL). |
```

Comment:
```suggestion
| CodeCommitRepo  | Optional  | CodeCommit repository name (just the name, not the full URL). |
```

Reply:
@ferdingler One docs miss, then it's good to go!

Reply:
fixed 😅